realm/realm-jni/src/util.cpp

```diff
@@ -185,21 +185,6 @@ jclass GetClass(JNIEnv* env, const char* classStr)
     return myClass;
 }
 
-void jprint(JNIEnv *env, char *txt)
```

Comment:
We are using the `TR` macros for logging. It will make it easier to support non-Android.